src/hotspot/share/gc/shared/collectedHeap.cpp

```diff
@@ -60,6 +60,7 @@
 
 class ClassLoaderData;
 
+Klass* CollectedHeap::_filler_klass = NULL;
```

Comment:
Please name this member `_filler_object_klass` to better distinguish it from `_filler_array*`.

Reply:
> Please name this member `_filler_object_klass` to better distinguish it from `_filler_array*`.

Updated.
Thanks.